azpubsub/contextvalidator/src/main/scala/kafka/security/auth/AzPubSubAclAuthorizer.scala

```diff
@@ -107,28 +107,34 @@ class AzPubSubAclAuthorizer extends Authorizer with KafkaMetricsGroup {
     */
   override def authorize(session: Session, operation: Operation, resource: Resource): Boolean = {
 
-    LOGGER.info(s"principal: ${Try(session.principal.getName).getOrElse("Empty principal name")}, Operation: ${operation.name}")
+    LOGGER.info(s"principal: ${Try(session.principal.getName).getOrElse("Empty principal name")}, Operation: ${operation.name}, Client Address: ${session.clientAddress.getHostAddress}")
 
-    newGauge(
-      AzPubSubAclAuthorizer.AzPubSubAclAuthorizingRequest,
-      new Gauge[Int] {
-        def value = 1
-      })
+    val meterAclAuthorizationRequest = newMeter(AzPubSubAclAuthorizer.AzPubSubAclAuthorizingRequest,
```

Comment:
Be aware that Gauge and Meter are two quite different metrics. One is absolute value, and one is rate.

Reply:
Discussed this offline. Don't need to change anything here.

Reply:
It looks  you create a new perf count object (val meterAclAuthorizationRequest = newMeter(AzPubSubAclAuthorizer.AzPubSubAclAuthorizingRequest) each time when this authorize method is called. Can you reuse one perf count object (e.g., make it static) since authorize method is called per every request.